examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -753,63 +754,49 @@ class WebGPUBackend extends Backend {
 
 	_getDepthBufferGPU( renderContext ) {
 
-		const { depthBuffers } = this;
 		const { width, height } = this.getDrawingBufferSize();
 
-		let depthTexture = depthBuffers.get( renderContext );
+		const depthTexture = this.defaultDepthTexture;
```

Comment:
Should we clone it here? Or is this unneeded?

Reply:
It's unneeded... I thought to add a cache to stencil/depth variations soon if needed too.